src/manifest.chrome.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Toggl Plan: Project Planning Calendar",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
I'd bump this in a separate commit on master / release PR.

Reply:
Noted.